third_party/Makefile

```diff
@@ -29,3 +32,11 @@ bandwidthTest: sb_micro_path
 	git clone -b v$(shell nvcc --version | grep 'release' | awk '{print $$6}' | cut -c2- | cut -d '.' -f1-2) https://github.com/NVIDIA/cuda-samples.git ./cuda-samples
 	cd ./cuda-samples/Samples/bandwidthTest && make clean && make TARGET_ARCH=x86_64 SMS="70 75 80 86"
 	cp -v ./cuda-samples/Samples/bandwidthTest/bandwidthTest $(SB_MICRO_PATH)/bin/
+
+# Build nccl-tests.
+# The version we use is commit 44df0bf from master branch, due to it didn't update release tag for long time.
+nccl_tests: sb_micro_path
+ifneq (,$(wildcard nccl-tests/Makefile))
+	cd ./nccl-tests && make MPI=1 MPI_HOME=$(shell whereis mpi | grep 'mpi' | awk '{print $$2}') -j
```

Comment:
there are multiple mpi in the image, pls make it configurable, e.g., add `MPI_HOME ?= /usr/local/mpi` line at the beginning